doc/eggex.md

```diff
@@ -311,6 +311,21 @@ Flags and translation preferences together:
 
     / digit+ ; ignorecase ; python /  # could translate to (?i)\d+
 
+The following flags are currently supported:
+
+#### `i` / `reg_icase` (Ignore Case)
+
+**With** this flag, the match _is not_ case sensitive.
```

Comment:
I think it can be more readable to give an example, like

> Use this flag to ignore case when matching.  For example `/'foo'; i/` matches 'FOO', but `/'foo'/` doesn't.

It's also shorter -- I think the two sentences here are a bit redundant